exercises/parallel-letter-frequency/README.md

```diff
@@ -0,0 +1,27 @@
+# Parallel Letter Frequency
+
+Count the frequency of letters in texts using parallel computation.
+
+Parallelism is about doing things in parallel that can also be done
+sequentially. A common example is counting the frequency of letters.
+Create a function that returns the total frequency of each letter in a
+list of texts and that employs parallelism.
+
+# Hints
```

Comment:
This should probably be added in a `hints.md` file in the .meta directory as well so that if the readme is regenerated it will generate this bit too :) (see the [contribution doc](https://github.com/exercism/java/blob/master/CONTRIBUTING.md#adding-a-new-exercise) for details on how to generate the readme)

Reply:
@FridaTveit the docs are a little unclear to me.  

1. Do I simply add the `hints.md` at the `.meta` root?  This seems to be what was done in EG pig-latin.
2. I have changed the `README.md` to have a "Java Tips" section instead of a "Hints" section.  Also, I have left the text in `README.md`, as I was unable to get the configlet tool to work after a couple of attempts.

I assumed the answer to both of the was 'yes'.  If not, let me know and I may need some guidance on additional required changes.

Reply:
The answer to 1 is definitely yes :) For 2, I think that should be fine. I assume doing it manually is fine if you can't get `configlet` to work, @stkent?

Reply:
I think that's fine; the next time we run an all-track update it will get refreshed :)